From 54b49a19f57ca630c52c74d065d0f3fc416a0116 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Wed, 23 Aug 2017 17:51:23 +0300
Subject: [PATCH 515/706] staging: fsl-dpaa2/mac: Comply with mc bus header
 updates

The fsl-mc bus driver reorganized some of its headers, so
update our includes accordingly.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
(cherry picked from commit b6064b3da2256d3c9ff58ca18c183d87e11309e6)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/dpmac.c | 3 +--
 drivers/staging/fsl-dpaa2/mac/mac.c   | 1 -
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/dpmac.c b/drivers/staging/fsl-dpaa2/mac/dpmac.c
index f7827423329e..baef769834ab 100644
--- a/drivers/staging/fsl-dpaa2/mac/dpmac.c
+++ b/drivers/staging/fsl-dpaa2/mac/dpmac.c
@@ -29,8 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#include "../../fsl-mc/include/mc-sys.h"
-#include "../../fsl-mc/include/mc-cmd.h"
+#include "../../fsl-mc/include/mc.h"
 #include "dpmac.h"
 #include "dpmac-cmd.h"
 
diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index ee28c3af6640..e9c1af8ab3b3 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -47,7 +47,6 @@
 #include <linux/phy_fixed.h>
 
 #include "../../fsl-mc/include/mc.h"
-#include "../../fsl-mc/include/mc-sys.h"
 
 #include "dpmac.h"
 #include "dpmac-cmd.h"
-- 
2.17.1

